src/blocks/time.rs

```diff
@@ -86,7 +88,13 @@ pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
 
         widget.set_values(map!(
             "icon" => Value::icon(api.get_icon("time")?),
-            "timestamp" => Value::datetime(Utc::now(), timezone.copied())
+            "timestamp" => {
+        let timezone: Option<Tz> = None; // or parse the timezone from a string
+        let datetime: DateTime<Utc> = Utc::now();
+        let format: Option<&str> = Some("%a %e{S} %B, %H:%M");
+
+        Value::datetime(datetime, timezone, Some(format))
```

Comment:
The third Parameter would be Option<Option<&str>>, which probably isn't what you want.